modules/space/rendering/renderableorbitalkepler.cpp

```diff
@@ -130,15 +199,22 @@ namespace {
         // [[codegen::verbatim(SegmentQualityInfo.description)]]
         int segmentQuality;
 
-        // [[codegen::verbatim(LineWidthInfo.description)]]
-        std::optional<float> lineWidth;
+        // [[codegen::verbatim(TrailWidthInfo.description)]]
+        std::optional<float> trailWidth;
 
-        // [[codegen::verbatim(LineColorInfo.description)]]
+        // [[codegen::verbatim(ColorInfo.description)]]
         glm::dvec3 color [[codegen::color()]];
 
         // [[codegen::verbatim(TrailFadeInfo.description)]]
         std::optional<float> trailFade;
 
+        enum class RenderingMode {
+            Trail,
+            Point
+        };
+        // [[codegen::verbatim(RenderingModeInfo.description)]]
+        std::optional<RenderingMode> renderingMode [[codegen::key("Rendering")]];
```

Comment:
Why the `codegen::key`?  Could we just name it `RenderableMode` in the asset instead?

Reply:
I think I did this in order to keep it aligned with `renderableTrail`.
We can change this to `RenderableMode`, `RenderingMode` or `RenderModede`.
If we change for this class, should we also change for `renderableTrail`?

Reply:
Ah.. keeping it aligned is good enough for me :+1: